js/Macros.js

```diff
@@ -12,11 +12,10 @@ function invokeMacro(place,macro,params,wikifier,tiddler)
 			window.tiddler = tiddlerElem ? store.getTiddler(tiddlerElem.getAttribute("tiddler")) : null;
 			window.place = place;
 			var allowEval = true;
-			if(config.evaluateMacroParameters=="system") {
-				if(!tiddler || tiddler.tags.indexOf("systemAllowEval") == -1) {
+			if(config.evaluateMacroParameters=="system")
+				if(!tiddler || tiddler.tags.indexOf("systemAllowEval") == -1)
 					allowEval = false;
-				}
-			}
+			
```

Comment:
imo same as above. We don't really win something here . Also removing the braces make debugging harder. Where to put `console.log()` if you need it inside the if statement!